test/wall_bc_tests.jl

```diff
@@ -191,11 +191,12 @@ function run_test(test_input, expected_phi, tolerance; args...)
         adv_input = advection_input("default", 1.0, 0.0, 0.0)
 		nrank_per_block = 0 # dummy value
 		irank = 0 # dummy value
-		comm = MPI.COMM_NULL # dummy value 
+		comm = MPI.COMM_NULL # dummy value
+        element_spacing_option = "uniform"
```

Comment:
Would be good to add sqrt spacing to this test too. This one wouldn't check correctness, but acts as a regression check that we do not break the sqrt grid spacing by mistake.

Above, add something like
```julia
test_input_chebyshev_sqrt_grid = merge(test_input_chebyshev,
                                       Dict("run_name" => "chebyshev_pseudospectral_sqrt_grid",
                                            "z_element_spacing_option" => "sqrt")
```

And after the "Chebyshev" test below, add
```julia
        @testset "Chebyshev" begin
            run_test(test_input_chebyshev_sqrt_grid,
                     [0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
                     2.e-3)
```
when you run the test, the failure message should show what the computed values were, which you can then paste in instead of the 0.0's.

You'll also need to add `"z_element_spacing_option" => "uniform"` to the `test_input_finite_difference` Dict above.